src/bin/cli.rs

```diff
@@ -10,6 +10,24 @@ use command_prelude::*;
 
 pub fn main(config: &mut Config) -> CliResult {
     let args = cli().get_matches_safe()?;
+
+    if args.value_of("unstable-features") == Some("help") {
+        println!(
+            "
+Available unstable (nightly-only) flags:
+
+    -Z avoid-dev-deps    -- Avoid installing dev-dependencies if possible
+    -Z minimal-versions  -- Install minimal dependency versions instead of maximum
+    -Z no-index-update   -- Do not update the registry, avoids a network request for benchmarking
+    -Z offline           -- Offline mode that does not perform network requests
+    -Z print-im-a-teapot -- Example option for demonstration purposes
```

Comment:
Teapot is just for testing the Cargo itself, I think we should not list it here :-)